test/integration/ctl/ctl_create_cr_test.go

```diff
@@ -60,8 +50,12 @@ func TestCtlCreateCR(t *testing.T) {
 		ns1     = "testns-1"
 		ns2     = "testns-2"
 
-		testdataPath = "../testdata/"
+		testWorkingDirectory, err = os.Getwd()
+		testdataPath              = testWorkingDirectory + "/testdata/"
 	)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Hm, not sure if this is in the right place 👀  `err` hasn't be reassigned since the last time it was checked from what I can see

Reply:
Yeah it's kinda hidden, on line 53 `testWorkingDirectory, err = os.Getwd()`

Reply:
Ah yeah, I see that now 😄 can you move that declaration out of the big `var` block to make it clear that this is the case? I can definitely see this checking getting refactored away at some point because someone misses that 🙈 better to use a few extra lines of code and make it clear 😄